core-java-8/pom.xml

```diff
@@ -1,200 +1,206 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
Looks like your editor may have changed the pom from spaces to tabs.

Let's change this back so that only the changes necessary for your article are present.